maplibre/src/io/apc.rs

```diff
@@ -41,18 +39,67 @@ pub enum Input {
     NotYetImplemented, // TODO: Placeholder, should be removed when second input is added
 }
 
+#[derive(Debug)]
+pub enum SendError {
+    // Error occured during transmission
+    Transmission,
+}
+
+impl Display for SendError {
```

Comment:
this seems weird, but works :)   Do you really need a Display trait on an error (here and in another trait)

Reply:
I will replace all manual implementations with thiserror derives: https://crates.io/crates/thiserror

Reply:
Doing this also in this PR